### PR TITLE
Add CACHE Gold token support

### PR DESCRIPTION
**Pull request by conscott**
CGT https://cache.gold/

### CODE REVIEW

File: server_extensions_extra/src/main/resources/batm-extensions.xml [hunk 1]
```diff
@@ -173,6 +174,7 @@
                 <cryptocurrency>BTDX</cryptocurrency>
                 <cryptocurrency>BTX</cryptocurrency>
                 <cryptocurrency>BURST</cryptocurrency>
+                <cryptocurrency>CGT</cryptocurrency>
```
Comment (pvyhnal-generalbytes) [inline server_extensions_extra/src/main/resources/batm-extensions.xml:177]: This also needs to be added to CoinmarketcapRateSource class

File: server_extensions_extra/src/main/resources/batm-extensions.xml [hunk 2]
```diff
@@ -406,6 +408,7 @@
             <cryptocurrency>BTDX</cryptocurrency>
             <cryptocurrency>BTX</cryptocurrency>
             <cryptocurrency>BURST</cryptocurrency>
+            <cryptocurrency>CGT</cryptocurrency>
```
Comment (pvyhnal-generalbytes) [inline server_extensions_extra/src/main/resources/batm-extensions.xml:411]: This also needs to be added to CoinGeckoRateSource class

**Comment by pvyhnal-generalbytes** [comment]
Please use template.svg as a base for your logo to keep the unified BATM design


SVG logos must comply with following rules:
1. SVG logos should not contain filters or bitmap images
2. All SVG and PNG logos should be 120 x 120px. See other logos for examples.
3. Preserve the unified BATM design: every SVG logo must contain the background from <a href="https://github.com/GENERALBYTESCOM/batm_public/blob/master/server_extensions_extra/src/main/resources/template.svg">template.svg</a>. An example background can be found <a href="https://github.com/GENERALBYTESCOM/batm_public/blob/master/server_extensions_extra/src/main/resources/lisk.svg">here</a>.

**Comment by generalbytes** [comment]
This pull request will be closed without merge due to author's inactivity.

**Pull request closed without merging**